.travis.yml

```diff
@@ -13,7 +13,7 @@ branches:
 cache:
   directories:
   - node_modules/
-script:  yarn travis
+script: bash test.sh
```

Comment:
As long as test.sh has a shebang (`#!/bin/bash`), we don't need to invoke this with bash.